### ISSUE TITLE
train_single_block_indices Parameter Appears Ineffective

### ISSUE THREAD

**Issue by Brackarnwar** [kohya-ss/sd-scripts#1970]
Command used:

accelerate launch --mixed_precision bf16 --num_cpu_threads_per_process 1 flux_train_network.py --pretrained_model_name_or_path "E:\0 - IA\ComfyUI2406\ComfyUI\models\unet\flux1-dev-fp8.safetensors" --clip_l "E:\0 - IA\ComfyUI2406\ComfyUI\models\clip\clip_l.safetensors" --t5xxl "E:\0 - IA\ComfyUI2406\ComfyUI\models\clip\t5xxl_fp8_e4m3fn.safetensors" --ae "E:\0 - IA\ComfyUI2406\ComfyUI\models\vae\ae.sft" --cache_latents_to_disk --save_model_as safetensors --sdpa --persistent_data_loader_workers --max_data_loader_n_workers 2 --seed 42 --gradient_checkpointing --mixed_precision bf16 --save_precision bf16 --network_module networks.lora_flux --network_dim 4 --network_train_unet_only --network_args "train_double_block_indices=none" "train_single_block_indices=7" --optimizer_type adamw8bit --learning_rate 1e-4 --cache_text_encoder_outputs --cache_text_encoder_outputs_to_disk --fp8_base --highvram --max_train_epochs 4 --save_every_n_epochs 1 --dataset_config "B:\kohya-ss\kohya_ss\sd-scripts\dataset_margot.toml" --output_dir "B:\kohya-ss\kohya_ss\sd-scripts\ouput" --output_name flux-lora-name --timestep_sampling shift --discrete_flow_shift 3.1582 --model_prediction_type raw --guidance_scale 1.0

Explanation Request:

    Why does specifying train_single_block_indices=7 not reduce training time compared to training all blocks?
    Could there be a conflict with train_double_block_indices=none or an issue in argument parsing?

**Comment by rockerBOO** [comment]
What is the training speed? There will be a baseline training speed due to the forward pass.
